src/main/java/org/amalitech/ContactBook/Main.java

```
@@ -0,0 +1,46 @@
+package org.amalitech.ContactBook;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+public class Main implements Serializable {
+    public static void main(String[] args) throws Exception {
```

The main method should throw an exception